src/style-spec/reference/v8.json

```
@@ -3696,10 +3697,10 @@
           "zoom"
         ]
       },
-      "doc": "The near and far limits of the fog layer. Units are relative to the map height, so that 2.0 corresponds to twice the map height, in pixels.",
+      "doc": "The near and far limits of the fog layer. Units are relative to the map height in CSS `vh` units, so that 200 corresponds to 200% the map height.",
```

Small edit to address this comment: https://github.com/mapbox/mapbox-gl-js/pull/10564#discussion_r621391997

```suggestion
      "doc": "The near and far distances in which fog fades within. Units are relative to the map window height in CSS `vh` units, so that 200 corresponds to 200% the map window height.",
```

I also replaced `map height` to mention `window`. We could also use `viewport`, but it seems more common in computer graphics/gl language. I don't think simply using `viewport height` or `window height` as a good option as it may suggest the entire page height instead, so leaving `map` seems appropriate.

---

An extra "within", but otherwise 👍 

```suggestion
      "doc": "The near and far distances in which fog fades. Units are relative to the map window height in CSS `vh` units, so that 200 corresponds to 200% the map window height.",
```